server/prometheus.js

```diff
@@ -34,6 +34,7 @@ class Prometheus {
      * @returns {Promise<void>}
      */
     static async init() {
+        PrometheusClient.register.clear();
```

Comment:
`Prometheus.init()` is only called once; here at startup:

https://github.com/louislam/uptime-kuma/blob/2a687ab67aa71b3cb36167f9eea7f5d77da32641/server/server.js#L201

Am I missing something?

Reply:
No, the function is only ever called once.

Why are you resetting things at the start of the function?
Why are you making it resistant to being called multiple times?